lib/src/presentation/view/card_detail.dart

```diff
@@ -190,4 +184,15 @@ class _CardDetailState extends State<CardDetail> {
       ),
     );
   }
+
+  dynamic isNotNull(cardParam, [text]) {
+    if (cardParam != null) {
+      return CardText(
+        text: text ?? cardParam,
+        isTitle: text != null ? Constants.titleIsTrue : Constants.titleIsFalse,
+      );
+    } else {
+      return const SizedBox.shrink();
+    }
+  }
```

Comment:
A few comments here, why you choose to return a dynamic? Also, this can be abstracted to a new widget. And let give it a more representative name

Reply:
Done